pkg/detectors/zulipchat/zulipchat_test.go

```diff
@@ -19,13 +20,13 @@ import (
 func TestZulipChat_FromChunk(t *testing.T) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
 	defer cancel()
-	testSecrets, err := common.GetSecret(ctx, "trufflehog-testing", "detectors3")
+	testSecrets, err := common.GetSecret(ctx, "trufflehog-testing", "detectors2")
```

Comment:
Now it's been changed to `detectors2`. `detectors3` was, and still is, correct - you can just leave it the way it was.